lib/clearance/password_strategies/sha1.rb

```diff
@@ -3,13 +3,21 @@ module PasswordStrategies
     module SHA1
       require 'digest/sha1'
 
+      DEPRECATION_MESSAGE = "[DEPRECATION] The SHA1 password strategy " \
+        "has been deprecated and will be removed from Clearance 2.0. BCrypt " \
+        "is the only officially supported strategy, though you are free to " \
+        "provide your own. To continue using this strategy add " \
+        "clearance-deprecated_password_strategies to your Gemfile."
```

Comment:
this is a little thing, but is there a reason why this gem has dashes and underscores in the name? Seems unusual.